### PR TITLE
Feature/uniswap example

### PR DESCRIPTION
**Pull request by taylorjdawson**
### Checklist
- [ ] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [ ] The box that allows repo maintainers to update this PR is checked
- [ ] I tested locally to make sure this feature/fix works
- [ ] I have run `yarn check` & `yarn build` to confirm there are not any associated errors
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -40,13 +44,13 @@
...
-    "@sveltejs/adapter-static": "^1.0.0-next.39",
-    "@sveltejs/adapter-vercel": "next",
     "unplugin-icons": "^0.13.4"
   },
   "type": "module",
   "dependencies": {
+    "@uniswap/widgets": "^2.7.0",
     "@web3-onboard/core": "^2.7.0",
-    "@web3-onboard/injected-wallets": "^2.0.16"
+    "@web3-onboard/injected-wallets": "^2.0.16",
```
Comment (Adamj1232) [inline docs/package.json:53]:
```suggestion
    "@web3-onboard/injected-wallets": "^2.2.3",
```
For latest wallet support and stability

File: docs/package.json [hunk 2]
```diff
@@ -14,11 +14,15 @@
     "format": "prettier --write --plugin-search-dir=. ."
   },
   "devDependencies": {
+    "@fontsource/ibm-plex-mono": "^4.5.11",
```
Comment (gesquinca) [inline docs/package.json:17]: Where are we using this?

Comment (taylorjdawson) [reply docs/package.json]: removed

**Review by techgangboss — APPROVED** [review]
looks good!

**Pull request merged**